app/scraping.py

```diff
@@ -5,15 +5,22 @@
 from io import BytesIO
 import logging
 
-# Define global variables for minimum image dimensions
-MIN_IMAGE_WIDTH = int(os.getenv("MIN_IMAGE_WIDTH", 300))
-MIN_IMAGE_HEIGHT = int(os.getenv("MIN_IMAGE_HEIGHT", 250))
+# Define global variables for allowed image dimensions
+ALLOWED_IMAGE_SIZES = os.getenv("ALLOWED_IMAGE_SIZES", "300x250,728x90,160x600,300x600").split(',')
 
 # Configure logging
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 
+def is_allowed_size(width, height):
```

Comment:
Use type hints

Reply:
Thank you for the feedback! I'll add type hints to the `is_allowed_size` function. I'll make the necessary changes and push an update shortly.